nano/node/active_transactions.cpp

```diff
@@ -257,22 +252,9 @@ void nano::active_transactions::cleanup_election (nano::unique_lock<nano::mutex>
 {
 	debug_assert (lock_a.owns_lock ());
 
-	if (!election->confirmed ())
-	{
-		node.stats.inc (nano::stat::type::election, nano::stat::detail::election_drop_all);
-		if (election->behavior () == election_behavior::hinted)
-		{
-			node.stats.inc (nano::stat::type::election, nano::stat::detail::election_hinted_drop);
-		}
-	}
-	else
-	{
-		node.stats.inc (nano::stat::type::election, nano::stat::detail::election_confirmed_all);
-		if (election->behavior () == election_behavior::hinted)
-		{
-			node.stats.inc (nano::stat::type::election, nano::stat::detail::election_hinted_confirmed);
-		}
-	}
+	node.stats.inc (nano::stat::type::active, nano::stat::detail::all, nano::stat::dir::out);
+	node.stats.inc (nano::stat::type::active, nano::to_stat_detail (election->behavior ()), nano::stat::dir::out);
```

Comment:
I believe `node.stats.inc (nano::stat::type::active, nano::to_stat_detail (election->behavior ()), nano::stat::dir::out);` will automatically increase `nano::stat::detail::all` all too for `nano::stat::type::active`, so the duplication might not be necessary.